projects/vdk-core/src/vdk/internal/builtin_plugins/config/log_config.py

```diff
@@ -57,12 +64,21 @@ def configure_loggers(
         "stream": "ext://sys.stderr",
     }
 
+    _SYSLOG_HANDLER = {
+        "level": "DEBUG",
+        "class": "logging.handlers.SysLogHandler",
+        "formatter": "detailedFormatter",
+        "address": log_endpoint,
+        "socktype": socket.SOCK_DGRAM,
```

Comment:
This should be configurable. Some syslog servers work only with TCP , some prefer UDP. We should not hard-code this.

Reply:
socket type is now configurable